src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  );
+};
```

Comment:
Please use destructuring for the user values
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
  const { email, name } = user;

  return (
    <a className="UserInfo" href={`mailto:${email}`}>
      {name}
    </a>
  );
};
```